packages/query/src/block-processor.ts

```diff
@@ -423,7 +423,7 @@ export class BlockProcessor implements BlockProcessorInterface {
     endHeightOfPreviousEpoch: bigint,
     latestKnownBlockHeight: bigint,
   ): Promise<void> {
-    const { sctParams } = await this.querier.app.appParams();
+    const { sctParams } = (await this.indexedDb.getAppParams()) ?? {};
```

Comment:
I also fixed this, we don't need to request `sctParams` from the full node since we store them in indexed-db

Reply:
@grod220 why's that?

Reply:
The params are used further down. There seems to be code in between that doesn't use the params.